src/Components/CriticalCareRecording/types/HemodynamicParametersType.res

```diff
@@ -0,0 +1,32 @@
+type rhythmVar = 
+| None
+| Regular
+| IrRegular
+
+type t = {
+  bp_systolic: string,
+  bp_diastolic: string,
+  pulse: string,
+  temperature: string,
+  respiratory_rate: string,
+  rhythm: rhythmVar,
+  description: string,
+}
+
+let bp_systolic= t => t.bp_systolic
+let bp_diastolic= t => t.bp_diastolic
+let pulse= t => t.pulse
+let temperature= t => t.temperature
+let respiratory_rate= t => t.respiratory_rate
+let rhythm= t => t.rhythm
+let description= t => t.description
+
+let init = {
+  bp_systolic: "",
+  bp_diastolic: "",
+  pulse: "",
+  temperature: "",
+  respiratory_rate: "",
+  rhythm: None,
+  description: "",
+}
```

Comment:
We do not need an init, you may remove this... also, please prefix the types file with `CriticalCare__` and add the module to `CriticalCare__Types.res` file :)

Reply:
done